src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothSocket.cs

```diff
@@ -49,7 +49,19 @@ public interface IBluetoothServerSocket
         /// </remarks>
         /// <param name="data">The data to be sent.</param>
         /// <since_tizen> 3 </since_tizen>
+        [Obsolete("Deprecated since API level 7. Please use SendData(byte[] data) method.")]
         int SendData(string data);
+
+        /// <summary>
+        /// Method to send data over bluetooth socket
+        /// </summary>
+        /// <returns>The number of bytes written (zero indicates nothing was written).</returns>
+        /// <remarks>
+        /// The connection must be established.
+        /// </remarks>
+        /// <param name="data">The data to be sent.</param>
+        /// <since_tizen> 7 </since_tizen>
+        int SendData(byte[] data);
```

Comment:
It seem to be move to `IBluetoothClientSocket`. 
Because no need `SendData` API in `ServiceSocket`

And No body directly implement `IBluetoothServerSocket` 
(`BluetoothServerSocket` class does not implement `BluetoothServerSocket`)